src/videojs.analytic.omniture.js

```diff
@@ -658,9 +658,6 @@ function o_debug(msg) {
 			}
 			that._playheadTime = that._player.currentTime();
 			var _duration = that._player.duration();
-			if (that._player.mediainfo){
-				_duration = that._player.mediainfo.duration/1000;
-			}

```

Comment:
This is the main change for anyone else looking for it. I'm not familiar with the `player.mediainfo` workings though so can't say if this is the right move. @bcandrew any thoughts?

All the other white space changes look good to me.